youth-leading-the-way.html

```diff
@@ -0,0 +1,536 @@
+              <a class="nav-link" href="workshops.html">Workshops</a>
+            </li>
+            <li>
+              <a href="./blog">Blog</a>
+            </li>
+            <li>
+              <a class="page-scroll" href="contactus.html">Contact Us</a>
+            </li>
+            <li class="nav-button">
+              <a class="btn btn-seconday btn-sm" href="index.html#help">
+                How can we help?
+              </a>
+            </li>
+            <!-- <a href="index.html#help" class="btn btn-seconday btn-sm" style="margin-left: 12px; margin-top: 12px;">How can we help?</a> -->
+          </ul>
+        </div>
+        <!-- /.navbar-collapse -->
+      </div>
+      <!-- /.container-fluid -->
+    </nav>
+  
+      </div>
+
+      <!-- header section -->
+      <div class="first-div">
+        <div class="first-landing-div">
+          <h2 class="help-title main-title">YOUTH LEADING THE WAY</h2>
+          <button class="header-button">PROGRAM</button>
+        </div>
+        <img class="landing-header" src="./img/lsc1.png" alt="" />
+      </div>
+
+      <!-- second section -->
+      <div class="need-to-volunteer">
+        <h3>Do you want more volunteering experience?</h3>
+        <p class="landing-volunteer-details">
+          Helping Hands is running a new enhanced volunteer program running into
+          March 2024. In this program, you can be matched to a long-term
+          volunteer opportunity where you can gain valuable experience, skills,
+          and long-term connections that can help you with your career. This
+          program comes with extra financial support that is not covered in any
+          of Helping Hands’ other programs. For your participation in this
+          program, you can receive some recognition and compensation.
+        </p>
+      </div>
+
+      <!-- third section -->
+      <div class="third-section">
+        <div class="third-section-div">
+          <h2>
+            Youth Leading the Way Program (now till March 2024) entails...
+          </h2>
+          <ol class="organised-list">
+            <li>
+              Free volunteer matching services for 60 youth participants to
+              volunteer service placement(s) to fulfill 120 hours of community
+              involvement.
+            </li>
+            <li>
+              Training sessions and mentorship provided by Helping Hands (which
+              also counts towards the 120 hours) for youth participants
+            </li>
+            <li>
+              Financial aid provided to youth by Helping Hands help them
+              participate in this program.
+            </li>
+            <li>
+              Youth participants can gain valuable experience, recognition, and
+              long-term connections from this program
+            </li>
+          </ol>
+          <button class="third-section-btn">
+            For any questions, please email outreach@helpinghandsapp.com
+          </button>
```

Comment:
Don't use the button tag if it doesn't do anything when clicked. If you want the aesthetic of a button, but it doesn't do anything, use div.